metabase/metabase
# Add database syncing status

Epic: https://github.com/metabase/metabase/issues/18675

A different approach to add a database syncing indicator that doesn't overlap with other elements.

<img width="1677" alt="Screen Shot 2021-12-13 at 9 35 15 am" src="https://user-images.githubusercontent.com/8542534/145763914-5ac58cf3-1f11-4a04-92b3-dbecd6a1b3d4.png">

<img width="1680" alt="Screen Shot 2021-12-13 at 9 35 02 am" src="https://user-images.githubusercontent.com/8542534/145763909-ee205edc-ae19-4566-a829-3141b1953660.png">

How to test:
- Go to Admin > Database > New database
- Add a new H2 database (you can copy connection details from Sample Dataset)
- You should see the indicator in the bottom right corner
- It should be visible in other parts of the app
